.github/workflows/ansible-test.yml

```diff
@@ -109,7 +109,9 @@ jobs:
       # OPTIONAL If your unit test requires Python libraries from other collections
       # Install them like this
       - name: Install collection dependencies
-        run: ansible-galaxy collection install ansible.netcommon -p .
+        run: |
+          ansible-galaxy collection install ansible.netcommon -p .
+          ansible-galaxy collection install ansible.utils -p .
```

Comment:
Does anything prevent us from adding the new package right into the first command invocation? It's usually better to feed everything into the same dependency resolution process.

Reply:
I always forget that `ansible-galaxy collection install` actually allows for that :)

Reply:
Fixed in 216b1e354411c4f4065d70d95a2118ffada489e0.